recipe/meta.yaml

```diff
@@ -46,13 +42,12 @@ requirements:
     - pkg-config    # [osx]
   host:
     - python
-    - setuptools
     - cython
     - pybind11
     - pythran
-    # this setting is actually an issue of the numpy version!!
-    - numpy-devel  {{ numpy }}  # [not (osx and arm64) and py<=39]
-    - numpy {{ numpy }}         # [(osx and arm64) or py>39]
+    - setuptools <60
+    - wheel
+    - numpy
```

Comment:
`numpy>=1.16.5,<1.24.0`
https://github.com/scipy/scipy/blob/v1.7.3/pyproject.toml#L55

Reply:
We use a matrix of `numpy` from the general `cbc.yaml`